src/modules/vtol_att_control/standard.cpp

```diff
@@ -363,6 +363,19 @@ void Standard::update_mc_state()
 		return;
 	}
 
+	// Do not engage pusher assist during a failsafe event
+	// There could be a problem with the fixed wing drive
+	if (_attc->get_vtol_vehicle_status()->vtol_transition_failsafe) {
+		return;
+	}
+
+	// disable pusher assist during landing
+	if (_attc->get_pos_sp_triplet()->current.valid
```

Comment:
Nevermind, navigator should clear it (as long as it's running). https://github.com/PX4/Firmware/blob/master/src/modules/navigator/navigator_main.cpp#L632